libbeat/docs/outputconfig.asciidoc

```diff
@@ -248,7 +248,7 @@ After the specified number of retries, the events are typically dropped.
 Some Beats, such as Filebeat, ignore the `max_retries` setting and retry until all
 events are published.
 
-Set `max_retries` to a value less than 0 to retry until all events are published. 
+Set `max_retries` to a value less than 0 to retry until all events are published.
```

Comment:
Instead of less then 0 I would specify -1. Less guesswork for the person configuring.